packages/web-app-files/src/components/SideBar/Shares/Collaborators/InviteCollaborator/InviteCollaboratorForm.vue

```diff
@@ -98,6 +98,10 @@ import {
   shareSpaceAddMemberHelp
 } from '../../../../../helpers/contextualHelpers.js'
 
+const $gettext = (str) => {
```

Comment:
Please add the usual comment above: `// just a dummy function to trick gettext tools`